packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -208,7 +208,7 @@ export interface IContainerRuntimeBase extends IEventProvider<IContainerRuntimeB
 	 */
 	orderSequentially(callback: () => void): void;
 
-	detachHead?(): { merge(): void; pause(): Promise<void>; dispose(): void };
+	enterStagingMode?(): { commitChanges(): void; discardChanges(): void };
```

Comment:
For prototype purposes might as well make this non-optional too (otherwise just an extra annoyance for integration).  Would also be nice to make a bound function for convenience (because IContainerRuntime is a big object that I don't want to hand out, I'll probably just want to hand out this method and it's [annoying to wrap like this](https://github.com/ChumpChief/FluidFramework/compare/test/detached-head-branching...ChumpChief:FluidFramework:IntegrateDetachedHeadPrototype?expand=1#diff-94cde51fe96914f27c7cc42db839d2309f83cd4c096624774af98a78f931fbb6R36-R38)).

```suggestion
	readonly enterStagingMode: () => {
		readonly commitChanges: () => void;
		readonly discardChanges: () => void;
	};
```